docker-for-mac/faqs.md

```diff
@@ -76,11 +76,33 @@ We recommend that you publish a port, or connect from another container. You can
 For more information and examples, see
 [I want to connect to a container from the Mac](networking.md#i-want-to-connect-to-a-container-from-the-mac) in the [Networking](/docker-for-mac/networking/) topic.
 
+### Can I use an SSH agent inside a container?
+
+Yes, you can use the host’s SSH agent inside a container. To do this:
+
+1. Bind mount the SSH agent socket by adding the following parameter to your `docker run` command:
+
+    `-v /run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock`
+
+1. Add the `SSH_AUTH_SOCK` environment variable in your container:
+
+    `-e SSH_AUTH_SOCK="/run/host-services/ssh-auth.sock"`
+
+To enable the SSH agent in Docker Compose, add the following flags to your service:
+
+ ```
+    volumes:
+      - /run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock
+    environment:
+      - SSH_AUTH_SOCK=/run/host-services/ssh-auth.sock
+ ```
```

Comment:
docker build --ssh should be mentioned as well?

Reply:
I couldn't make it work. Any idea how we can do it ?
This sock is only accessible by the daemon in the VM. 

```
docker build --ssh default=/run/host-services/ssh-auth.sock -t foo .
could not parse ssh: [default=/run/host-services/ssh-auth.sock]: stat /run/host-services/ssh-auth.sock: no such file or directory
```

Reply:
build --ssh is for the sock on the client

Reply:
Yes, where possible, we should also point people to the `--ssh` option for `docker build` (when using buildkit) to promote safe sharing of credentials. I think we still may have to add proper documentation for that though? https://github.com/docker/cli/pull/1419)

Reply:
Thanks for the link @thaJeztah, it works for me now :)
Maybe we can add a note that this is only required for old build system ?

Reply:
Looks like we need to double check the `build --ssh` option before documenting it as supported. Please let me know when this is complete and I'll update the docs.

Reply:
Created https://docker.atlassian.net/browse/ENGDOCS-326 to track this work item.